### ISSUE TITLE
CLI to get/set dask config values

### ISSUE THREAD

**Issue by ntabris** [dask/dask#9746]
Here's something I think would be nice, since it's often easier to tell people "run this CLI command" rather than "dig around in some YAML files looking for `foo` and change the value to `123`".

```
dask config get foo.bar  # prints value of foo.bar (or nothing if not set?)
dask config get  # prints entire dask config
dask config set foo.bar 123  # adds/updates foo.bar in the appropriate YAML file
```

`set` is trickier than get, because we'd have to pick the right YAML file to write to. Here's the logic I have in mind:

1. If key already is set in a config, then update value in highest priority config file where it's set.
2. If key isn't set then... my inclination is to have a default file (maybe `dask.yaml`, maybe something new like `extra-config.yaml`) in `~/.config/dask/` where this would go, but maybe also allow something like:

```
dask config set foo.bar 123 --file ~/.config/dask/my-desired-config-file.yaml
```

(or maybe you could just give file name and it would always be in `~/.config/dask`).

For bonus points, I think it would be handy to have

```
dask config find foo.bar
```

which would tell you all the different places (in config search paths) that `foo.bar` is set.

Note that this relates to/interacts with https://github.com/dask/dask/issues/9702.

**Comment by crusaderky** [comment]
Re-opening as the current implementation of set always writes to dask.yaml, whereas the ticket correctly points out that it would be better to write to the correct config file (distributed.yaml etc). This can create problems where the same config key appears, with differetn values, in multiple files inside `~/.config/dask/`.

**Comment by milesgranger** [comment]
Closed by #10930, please reopen if I'm mistaken. :)
